smac/runhistory/runhistory.py

```diff
@@ -326,18 +356,19 @@ def update_from_json(self, fn, cs):
         """
         new_runhistory = RunHistory(self.aggregate_func)
         new_runhistory.load_json(fn, cs)
-        self.update(runhistory=new_runhistory)
+        self.update(runhistory=new_runhistory, origin=DataOrigin.EXTERNAL_SAME_INSTANCES)
```

Comment:
the origin should maybe also here an argument of the method. I don't have a specific use case, but the code would be more flexible.